installing/README.md

```diff
@@ -0,0 +1,10 @@
+# Installing Nix
+
+Nix is a package manager not unlike pip, bundler, npm or
+bower. Installing Nix for a single user is very simple and works on
+almost every standard linux system.
+
+
+> **Note** <br>Nix installs itself in your root directory under `/nix` and can grow
+quite large. You should have 20 GiB of free space on your root
```

Comment:
that's completely dependant on what they install no? if they are only trying it out they wouldn't reach 20go